doc/tutorial/single_cell_recipe_result.svg

```diff
@@ -0,0 +1,691 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
If it builds exactly the same model then we get the same results and we can use the same [file](https://arbor.readthedocs.io/en/latest/_images/single_cell_model_result.svg) as the previous example.

Reply:
True, I'll change it.